x/staking/keeper/keeper_test.go

```diff
@@ -0,0 +1,27 @@
+package keeper_test
+
+import (
+	"testing"
+
+	"github.com/KuChainNetwork/kuchain/test/simapp"
+	"github.com/KuChainNetwork/kuchain/x/staking/types"
+	. "github.com/smartystreets/goconvey/convey"
+	abci "github.com/tendermint/tendermint/abci/types"
+)
+
+func TestKeeper(t *testing.T) {
+	wallet := simapp.NewWallet()
+	Convey("TestParams", t, func() {
+		_, _, _, _, _, _, app := NewTestApp(wallet)
+		keeper := app.StakeKeeper()
+		keeper = keeper.EmptyHooks()
+		ctx := app.BaseApp.NewContext(true, abci.Header{Height: app.LastBlockHeight() + 1})
+		expParams := types.DefaultParams()
+		resParams := keeper.GetParams(ctx)
+		So(expParams.Equal(resParams), ShouldBeTrue)
```

Comment:
So(expParams, ShouldBeEqual, resParams)

or 

So(expParams, simapp.ShouldEq, resParams) for chain types

Reply:
use require.Equal